src/api/java/baritone/api/utils/ExampleBaritoneControl.java

```diff
@@ -662,10 +716,47 @@ public boolean isInGoal(int x, int y, int z) {
         }
         if (msg.equals("damn")) {
             logDirect("daniel");
+            return true;
```

Comment:
no

Reply:
why?

Reply:
intended easter egg. typing `damn` in chat should still send `damn` it just has baritone also do `[Baritone] daniel`

Reply:
I'm not going to test that right now but when I tested it worked like this:
`#daim`
`[Baritone] daniel`
`[Baritone] Invalid command`

or something like that

Reply:
this is intended